GCRCatalogs/alphaq.py

```diff
@@ -12,6 +12,7 @@
 __all__ = ['AlphaQGalaxyCatalog']
 
 
+
```

Comment:
Add a line `__version__ = '2.1.1.1'` (or your preferred versioning scheme) to indicate the version of this reader file.